collector/container/rhtap-slim.Dockerfile

```diff
@@ -0,0 +1,151 @@
+# FROM registry.access.redhat.com/ubi8/ubi:latest AS builder
+FROM quay.io/centos/centos:stream9 AS builder
+
+COPY . .
+
+# TODO(ROX-20234): use hermetic builds when installing/updating RPMs becomes hermetic.
+RUN dnf -y update \
+    && dnf -y install --nobest \
+        autoconf \
+        automake \
+        binutils-devel \
+        bison \
+        ca-certificates \
+        clang-15.0.7 \
+        cmake \
+        cracklib-dicts \
+        diffutils \
+        elfutils-libelf-devel \
+        file \
+        flex \
+        gcc \
+        gcc-c++ \
+        gdb \
+        gettext \
+        git \
+        glibc-devel \
+        libasan \
+        libubsan \
+        libcap-ng-devel \
+        libcurl-devel \
+        libtool \
+        libuuid-devel \
+        make \
+        openssh-server \
+        openssl-devel \
+        patchutils \
+        passwd \
+        pkgconfig \
+        rsync \
+        tar \
+        unzip \
+        valgrind \
+        wget \
+        which \
+        bpftool \
+    && dnf clean all
+
+ARG BUILD_DIR
+ENV BUILD_DIR=${BUILD_DIR}
+
+ARG SRC_ROOT_DIR
+ENV SRC_ROOT_DIR=${SRC_ROOT_DIR}
+
+ARG CMAKE_BUILD_DIR
+ENV CMAKE_BUILD_DIR=${CMAKE_BUILD_DIR}
+
+# TODO(ROX-20240): CMAKE_BUILD_TYPE should probably not be Release for PR, normal branch builds
+ARG CMAKE_BUILD_TYPE=Release
+ENV CMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}
+
+# Appends an argument to the driver download URL that is used for filtering alerts on missing kernels.
+# TODO(ROX-20240): This needs to be true on PRs only.
+ARG COLLECTOR_APPEND_CID=false
+ENV COLLECTOR_APPEND_CID=${COLLECTOR_APPEND_CID}
+
+WORKDIR ${BUILD_DIR}
+
+RUN mkdir kernel-modules \
+    && cp -a /builder builder \
+    && cp -a /collector collector \
+    && cp -a /falcosecurity-libs falcosecurity-libs \
+    && cp -a /builder/third_party third_party \
+    && cp -a /kernel-modules/MODULE_VERSION kernel-modules/MODULE_VERSION \
+    && cp -a /CMakeLists.txt CMakeLists.txt
+
+# WITH_RHEL_RPMS controls for dependency installation, ie if they were already installed as RPMs.
+# Setting the value to empty will cause dependencies to be downloaded from repositories or accessed in submodules and compiled.
+# TODO(ROX-20651): Set ENV WITH_RHEL_RPMS=true when RHEL RPMs can be installed to enable hermetic builds.
+RUN ./builder/install/install-dependencies.sh \
+    && cmake -DDISABLE_PROFILING=ON -S "${SRC_ROOT_DIR}" -B "${CMAKE_BUILD_DIR}" \
+    && cmake --build "${CMAKE_BUILD_DIR}" --target all -- -j "$(nproc)" \
+    && (cd ${CMAKE_BUILD_DIR} && ctest -V) \
+    && strip --strip-unneeded "${CMAKE_BUILD_DIR}/collector/collector" "${CMAKE_BUILD_DIR}/collector/EXCLUDE_FROM_DEFAULT_BUILD/libsinsp/libsinsp-wrapper.so"
+
+FROM registry.access.redhat.com/ubi9/ubi-minimal:9.2
+
+ARG BUILD_TYPE=rhel
+ARG ROOT_DIR=.
+# TODO(ROX-20236): configure injection of dynamic version value when it becomes possible.
+ARG COLLECTOR_VERSION=0.0.1-todo
+ARG MODULE_VERSION=0.0.1-todo
+
+ENV ROOT_DIR=$ROOT_DIR
```

Comment:
I don't see `$ROOT_DIR` set in the downstream dockerfile. Are you sure it's needed?